pages/navigation/Navigation.js

```diff
@@ -125,6 +139,10 @@ function Navigation({ user } = props) {
             <small className="top display text-light">
               Signed in as: {user.email} ({role.toUpperCase()})
             </small>
+            <small hidden className="user-id">
+              user.id:{user.id}
+            </small>
```

Comment:
Curious what this is for?

Reply:
to later use in form for content submitter id